gamemodes/cinema/gamemode/modules/scoreboard/cl_playerlist.lua

```diff
@@ -203,6 +216,20 @@ function PLAYER:UpdatePlayer()
 			self.Mute:SetImage( "icon32/unmuted.png" )
 		end
     end
+	
+	self.ChatMute.DoClick = function()
+		if !IsValid(self.Player.IsChatMuted) then 
```

Comment:
still wrong: https://wiki.garrysmod.com/page/Global/IsValid

IsValid is only for entities, this is just a boolean/nil (NULL Entity and nil are not the same thing). Just do IsChatMuted = not IsChatMuted